xbmc/addons/addoninfo/AddonInfoBuilder.cpp

```diff
@@ -441,6 +441,15 @@ bool CAddonInfoBuilder::ParseXML(const AddonInfoPtr& addon, const TiXmlElement*
     }
   }
 
+  if (!isRepoXMLContent)
+  {
+    using XFILE::CFile;
+    if (CFile::Exists(URIUtils::AddFileToFolder(addonPath, "resources", "settings.xml")))
+      addon->m_supportBaseSettings = true;
```

Comment:
`m_supportAddonSettings`? we wanted to get rid of "base", because it is so generic that it means nothing to the code reader.